### DISCUSSION TITLE
Compression issue when logging in & update failing

### DISCUSSION THREAD

**Discussion by DeathManeYT-ModCode** [ppy/osu#22370] [Q&A]
### Type

Game behaviour

### Bug description

Every time I try to login to osu!i get an error message related to Brotli compression and the game won't update.

### Screenshots or videos

![osu! 1_21_2023 6_44_05 PM](https://user-images.githubusercontent.com/121408915/213966041-ffccc5a0-20fd-4744-98ec-94590ed6173b.png)

### Version

2023.123.0-lazer

### Logs

[performance-input.log](https://github.com/ppy/osu/files/10476782/performance-input.log)
[performance-update.log](https://github.com/ppy/osu/files/10476783/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/10476784/runtime.log)
[updater.log](https://github.com/ppy/osu/files/10476785/updater.log)
[database.log](https://github.com/ppy/osu/files/10476786/database.log)
[input.log](https://github.com/ppy/osu/files/10476787/input.log)
[performance.log](https://github.com/ppy/osu/files/10476788/performance.log)

**Comment by peppy** [comment]
Please try re-running the installer. It looks like some external application is fiddling with osu!'s DLLs (likely a virus scanner).
